config.py

```diff
@@ -1198,26 +1194,29 @@ def load_check_from_places(check_config, check_name, checks_places, agentConfig)
         # Validate custom checks and wheels without a `datadog_checks` namespace
         if not agentConfig.get("disable_py3_validation", False):
             if version_override in (UNKNOWN_WHEEL_VERSION_MSG, CUSTOM_CHECK_VERSION_MSG):
-                a7_compatible = A7_COMPATIBILITY_READY
+                py3_compatible = PY3_COMPATIBILITY_UNKNOWN
+                warnings = []
                 try:
                     file_path = os.path.realpath(check_path.decode(sys.getfilesystemencoding()))
-                    warnings = a7.validate.validate_py3(file_path)
+                    output, _, _ = get_subprocess_output(
+                        [sys.executable, "-m", "pylint", "-f", "json", "--py3k", "-d", "W1618", "--persistent", "no", "--exit-zero", file_path], log)
+                    warnings = json.loads(output)
```

Comment:
I think you need to add a condition here, because on old versions of pylint, the output can be '' instead of '[]' when there is no errors.

Reply:
thanks, addressed this. Note that `get_subprocess_output` raises `SubprocessOutputEmptyError` if the output is empty, and the logic handles this exception